tools/clang/test/CodeGenDXIL/hlsl/objects/HitObject/hitobject_make.hlsl

```diff
@@ -0,0 +1,75 @@
+// RUN: %dxc -T lib_6_9 -E main %s | FileCheck %s --check-prefix DXIL
+// RUN: %dxc -T lib_6_9 -E main %s -fcgl | FileCheck %s --check-prefix FCGL
+// RUN: %dxc -T lib_6_9 -E main %s -ast-dump-implicit | FileCheck %s --check-prefix AST
```

Comment:
I think AST tests belong under SemaHLSL.  We could probably address this later for these tests, but in the future, we should be putting them there.  For -fcgl and full compilation, that should go here, but we also need to add a `lit.local.cfg` under `CodeGenDXIL/hlsl/objects/HitObject` to require `dxil-1-9`.  This is not a requirement for SemaHLSL tests.

Reply:
Tracking this in #7359

Reply:
Added a `lit.local.cfg` to the `CodeGenDXIL/../HitObject` dir